src/norm/private/postgres/dbtypes.nim

```diff
@@ -21,7 +21,7 @@ func dbType*(T: typedesc[int16]): string = "SMALLINT"
 
 func dbType*(T: typedesc[int32]): string = "INTEGER"
 
-func dbType*(T: typedesc[int64 | Positive | int | Natural]): string = "BIGINT"
+func dbType*(T: typedesc[int64 | Positive | int | Natural | enum]): string = "BIGINT"
```

Comment:
@PhilippMDoerner I wonder if it's more appropriate to store enum as INTEGER and not BIGINT. You probably won't have a BIGINT number of enum values.

Reply:
Fair point, I agree with the reasoning. I changed it to use INTEGER instead. If somebody needs something else, they can just write their own dbType as that seems to me to be a special case.